lib/drivers/docker/index.js

```diff
@@ -0,0 +1,221 @@
+var sCtl = require('strong-control-channel/client');
+var Deployment = require('./deployment');
+var EventEmitter = require('events').EventEmitter;
+var fmt = require('util').format;
+var inherits = require('util').inherits;
+var sdr = require('strong-docker-run');
+var _ = require('lodash');
+
+module.exports = exports = DockerDriver;
+
+function DockerDriver(opts) {
+  if (!opts.baseDir || !opts.console || !opts.server) {
+    throw Error('Requires baseDir, console, and server');
+  }
+  if (!(this instanceof DockerDriver)) {
+    return new DockerDriver(opts);
+  }
+  EventEmitter.call(this);
+
+  // XXX(rmg) can I put temp files here?
+  this.baseDir = opts.baseDir;
+  // XXX(rmg) for log messages? or something else?
+  this.console = opts.console;
+  // XXX(rmg) this is pm's "Server" instance, right?
+  this.pm = opts.server;
```

Comment:
so, speaking of the above, I'd prefer this to be `this.server = opts.server`, name aliasing is confusing. Or go nuts, and change all the `.server` properties (direct driver, service-instance, etc) to `pm`... but when we use this driver in the strong-executor, `pm` may not seem so great a name, either.